lesson1.html

```diff
@@ -170,7 +170,7 @@ <h2>for each</h2>
         <section>
           <h2>Whiteboard Exercise</h2>
           <p>Write pseudocode for the following:</p>
-          <blockquote>Write out a receipt for each item in Santa’s sack of presents.</blockquote>
+          <blockquote>A hotel wants confirmation of all the bookings over the next week. Print each booking number from the list <code>thisWeekBookings</code>.</blockquote>
```

Comment:
I'm not sure how reveal js will treat this <code> block, it might not display inline.

Reply:
It worked before I thought? Can I test this?